roles/aci/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# This role contains tasks specific to aci
```

Comment:
This is really a matter of taste, analogous to breaking up of code modules/api's etc. Based on the things we do for aci, it seems to make sense to have a separate task. At least I can't see how that would hurt...